internal-distro.py

```diff
@@ -202,7 +203,7 @@ def main():
                 print('    Found package "{:s}" => {:s}'.format(
                     installed_package.name, installed_package.location))
 
-            installed_packages.update(repository_package_map.iterkeys())
+            installed_packages.update(iterkeys(repository_package_map))
```

Comment:
`.keys()` if you want to be extra explicit, but just passing `repository_package_map` should work automatically.

Reply:
iterkeys is redundant (again, as per PEP 469), but I see no reason to add more unnecessary syntatic sugar